server/fleet/hosts.go

```diff
@@ -20,6 +20,8 @@ const (
 	// StatusNew means the host has enrolled in the interval defined by
 	// NewDuration. It is independent of offline and online.
 	StatusNew = HostStatus("new")
+	// StatusMissing means the host is missing for 10 days.
```

Comment:
Fix comment, this should be `30 days`, yes?
Also might be a good idea to add a comment explaining why `StatusMissing` and `StatusMIA` seem like they are duplicated functionality